controllers/discordactions.js

```diff
@@ -122,8 +122,28 @@ const addGroupRoleToMember = async (req, res) => {
   }
 };
 
+/**
+ * Gets all group-roles
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const updateDiscordImageForVerification = async (req, res) => {
+  try {
+    const { id: userDiscordId } = req.params;
+    const discordAvatarUrl = await discordRolesModel.updateDiscordImageForVerification(userDiscordId);
+    return res.json({
+      message: "Discord avatar URL updated successfully!",
+      discordAvatarUrl,
+    });
+  } catch (err) {
+    logger.error(`Error while adding new Role: ${err}`);
```

Comment:
Nit : we can give message `error happen while updating` instead  of `adding` as this function is updating as per the function name.

Reply:
😂😂😂😂done